### ISSUE TITLE
No xbox360 controller in joy.cpl

### ISSUE THREAD

**Issue by zsohu** [Ryochan7/DS4Windows#53]
Hi,

From a longer break I wanted to use my DS4 over BT for playing some forza 3.  I had the old 1.4.52 version from Jays installed. After I did my usual thing before playing (staring DS4 win, turning on the controller, choosing profile), I noticed that forza is not responding to any inputs I made. After quitting forza I checked the joy control panel and there were no controllers listed at all if I'm using exclusive mode. If I disable exclusive mode only the "standard" wireless controller showing. If I plug in the controller to USB everything works as intended.

After looking some forums for solutions I found your work Ryochan7. I downloaded the most recent version (1.4.74) but the issue is the same. I'm on win10 with creators update.

What could cause this problem? If needed I'm more happy to provide any further details.

Thank you in advance!

Regards,
Zsolti

**Comment by Ryusennin** [comment]
I don't use Win10, but this warning from the main GitHub page might be relevant to you:

"Hide DS4 currently doesn't work on Windows 10 without a workaround (disabling Network List Service and restarting, but this causes serious issues with windows, use caution)."

**Comment by zsohu** [comment]
Thanks for the suggestion, but the issue is present whether or not "Hide DS4 checked".

**Comment by Ryochan7** [comment]
There have been related issues regarding interfacing with Scp Virtual Bus. Some issues have been dealt with but others have been harder to nail down especially when they are hard to reproduce on my system. Exclusive mode has had its fair share of problems and some of those issues have been addressed. It doesn't sound like the problem is related to exclusive mode since the big issue would be the controller getting permanently disabled. I use Bluetooth primarily and I unfortunately don't run into these kinds of problems on my system.

**Comment by zsohu** [comment]
The funny thing that DS4Windows is handling the controller because the touch pad is working and i can see the controller readouts, but nothing in the joy.cpl.

**Comment by zsohu** [comment]
Usually, for mystical issues like this, the underlying problem usually is some kind of a user error. The issue was that the "Use Dinput only" option was selected hence the program disabled the xbox360 emulation (facepalm). I don't remember why - maybe accidentally - I checked this before, but that is certain that after the break I took playing with my DS4 I definitely forget about it. Thanks for both of you trying to help me.

Keep up your good work, I really appreciate keeping this must have tool for us alive!
